cranelift/filetests/filetests/isa/aarch64/call.clif

```diff
@@ -850,16 +850,14 @@ block0(v0: i64):
 
 ; VCode:
 ; block0:
-;   mov x5, x8
-;   movz x4, #42
-;   str x4, [x8]
+;   movz x2, #42
+;   str x2, [x8]
```

Comment:
It's neat that this is actually removing moves in some cases compared to the older RA2!

Reply:
That's actually due to my change to `sret` register handling :sweat_smile: 

I'll land it as a separate PR once I've resolved all of the new move issues.